bin.src/make_remote_butler.py

```diff
@@ -117,6 +117,11 @@ def _hsc_rc2(butler):
             "HSC/calib/DM-28636",
         ],
     )
+
+    butler.registry.registerCollection(
+        instrument.makeUnboundedCalibrationRunName(),
+        type=CollectionType.CHAINED
+    )
```

Comment:
Maybe add a similar line above for the calibration collection, just in case?

Reply:
It wouldn't be needed because it's done already at https://github.com/lsst-dm/prompt_prototype/blob/ca1d53afe06bd239ac35211f4510781d0c23712d/bin.src/make_remote_butler.py#L95